OCP 4.18 Release Notes for P/Z (#86406)
openshift/openshift-docs: release_notes/ocp-4-18-release-notes.adoc

[id="ocp-release-notes-ibm-z_{context}"]
=== {ibm-z-title} and {ibm-linuxone-title}

With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see

> xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Preparing to install on {ibm-z-title} and {ibm-linuxone-title}].

Reviewer: ```suggestion
xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installation methods].
```
the actual title of the section

Reviewer 2: Agree we changed the title with the restructuring.

[IMPORTANT]
====
Compute nodes must run {op-system-first}.
====

[discrete]
[id="ocp-4-18-ibm-z-enhancements_{context}"]
==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements

The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.

> This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:

Reviewer 3: ```suggestion
This release introduces support for the following feature on {ibm-z-name} and {ibm-linuxone-name}:
```

- Use singular `feature`:  cause there is only one feature.

> * Adding compute nodes to on-premise clusters using `oc`

Reviewer: ```suggestion
* Adding compute nodes to on-premise clusters using {oc-first}
```
This looks like the first mention of the OpenShift CLI in the release notes as far as I can see by searching.

[id="ocp-release-notes-ibm-z_{context}"]
=== {ibm-z-title} and {ibm-linuxone-title}

With this release, {ibm-z-name} and {ibm-linuxone-name} are now compatible with {product-title} {product-version}. You can perform the installation with z/VM, LPAR, or {op-system-base-full} Kernel-based Virtual Machine (KVM). For installation instructions, see
xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Preparing to install on {ibm-z-title} and {ibm-linuxone-title}].

[IMPORTANT]
====
Compute nodes must run {op-system-first}.
====

[discrete]
[id="ocp-4-18-ibm-z-enhancements_{context}"]
==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements

The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.

This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:

> * Agent-based Installer day 2

Reviewer 2: ```suggestion
* Adding compute nodes to on-premise clusters
```

> * Adding compute nodes to on-premise clusters

Reviewer 2: ```suggestion
* Adding compute nodes to on-premise clusters using `oc`
```

Author: Updated as suggested above.

[id="ocp-release-notes-ibm-power_{context}"]
=== {ibm-power-title}

The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.

This release introduces support for the following features on {ibm-power-title}:

> //* Additional Hardware Configurations for ABI/AI/HCP on P

Reviewer 4: ```suggestion
//* Additional Hardware Configurations for ABI/AI/HCP on P
```

What does the slash mean?

Reviewer 2: The slash means it's commented out. At least for Z we decided not to mention this as it's not really a feature.

Reviewer 4: ack

Reviewer 4: I'm leaning towards commenting it out as you mention. I think it's too confusing to mention.

Author: Removed the commented line.

> * Agent-based Installer Post-Installation add workers

Reviewer 2: ```suggestion
* Adding compute nodes to on-premise clusters
```
@alishaIBM @prb112 I would use the wording x86 has used in the [4.17 RNs](https://docs.openshift.com/container-platform/4.17/release_notes/ocp-4-17-release-notes.html#ocp-4-17-adding-nodes-to-on-prem_release-notes)

The underlying technology utilizes the ABI, however users don't need to know how it's implemented. Any cluster on the supported platforms can use this method regardless of whether or not the cluster was installed using the ABI.

Reviewer 2: Also in the supported features table please add a new row before Agent-based Installer

```
|Adding compute nodes to on-premise clusters
|Supported
|Supported
```

Reviewer 4: Contextually, shouldn't it mention agent/assisted installers?

Reviewer 2: > Contextually, shouldn't it mention agent/assisted installers? I only noticed going back and reading it.

I would prefer staying with what @skopacz1 told me , that the user doesn't need to know how it's implemented.

Reviewer 5: FYI that this preference for just mentioning the functionality while leaving out the underlying tech comes from @andfasano, who originally implemented the node-adding feature

Reviewer 6: Since this is the way it's communicated for other platforms, it makes sense to me to communicate it without explicit note of the underlying tech. Also helps with not confusing it with the way the cluster was initially installed, which is not relevant for this feature.

Author: Thank you all. Updated as per the above discussion.

Reviewer 2: > FYI that this preference for just mentioning the functionality while leaving out the underlying tech comes from @andfasano, who originally implemented the node-adding feature

@prb112 see above

> * Adding compute nodes to on-premise clusters

Reviewer 4: ```suggestion
* Adding compute nodes to on-premise clusters
```

This existed in prior versions. I think the context is everything as to why clients would care. The Agent/Assisted context is very important here.

Reviewer 2: ```suggestion
* Adding compute nodes to on-premise clusters using `oc`
```

Author: Updated as suggested above.

xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Preparing to install on {ibm-z-title} and {ibm-linuxone-title}].

[IMPORTANT]
====
Compute nodes must run {op-system-first}.
====

[discrete]
[id="ocp-4-18-ibm-z-enhancements_{context}"]
==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements

The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.

This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:

* Agent-based installer day 2

[id="ocp-release-notes-ibm-power_{context}"]
=== {ibm-power-title}

The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.

This release introduces support for the following features on {ibm-power-title}:

//* Additional Hardware Configurations for ABI/AI/HCP on P
* Agent-based Installer day2

> * Compliance/SP Operator Support for Multi-Arch Compute Environments

Reviewer 2: @prb112 Will this be ready for the 4.18 GA or release async after GA? 

cc @werled

Reviewer 4: release async

|====
|Feature |{ibm-power-name} |{ibm-z-name} and {ibm-linuxone-name}

> |Adding compute nodes to on-premise clusters

Reviewer 4: It's always been supported. This is with the context of Agent/Assisted.
